app/_data/docs_nav_gateway_3.8.x.yml

```diff
@@ -392,6 +392,12 @@ items:
             url: /hub/kong-inc/ai-proxy/how-to/llm-provider-integration-guides/llama2/
             generate: false
             absolute_url: true
+      - text: AI Platform Integration Guides
+        items:
+          - text: Gemini
+            url: /hub/kong-inc/ai-proxy/how-to/ai-and-machine-learning-platform-integration-guides/gemini/
```

Comment:
```suggestion
            url: /hub/kong-inc/ai-proxy/how-to/machine-learning-platform-integrations/gemini/
```

Reply:
I believe you have to change this too, if you don't ignore this.

Reply:
@Guaris Yep, thanks for catching that!